### ISSUE TITLE
Should set a screen without opening the menu

### ISSUE THREAD

**Issue by egulhan** [RafaelBarbosatec/hidden_drawer_menu#27]
Hi.

In some cases, I should open some screens which are on the menu. If I open these screens using **Navigator.push()**, menu will not be shown. So, to handle this problem, I use **SimpleHiddenDrawerProvider.of(context).setSelectedMenuPosition()**. But everytime I use setSelectedMenuPosition(), it opens the menu. I should set screen without opening the menu for these kind of cases.

**Comment by nikkilr88** [comment]
I have this same problem. Is there any way to prevent the menu from opening?

**Comment by EbrahimMosaaed** [comment]
> I have this same problem. Is there any way to prevent the menu from opening?
i have the same issue, did you solve it ?

**Comment by egulhan** [comment]
> I have this same problem. Is there any way to prevent the menu from opening?

Hello.

Yes, I handled this problem.  First, you should create a menu widget (actually screen - MenuScreen for me). By this, you will show content with menu. You should call like below:
```
Navigator.push(
	context,
	MaterialPageRoute(builder: (context) {
	  return MenuScreen(
	    activeMenuItem: Menu.MENU_ITEM_INDEX_HOME,
	  );
	}),
);
```

**Comment by RafaelBarbosatec** [comment]
functionality available in the version 2.0.1.
`setSelectedMenuPosition(1,openMenu: false)`
